packages/eslint-plugin/src/rules/ban-ts-comment.ts

```diff
@@ -197,11 +198,10 @@ export default createRule<[Options], MessageIds>({
             if (directive === 'ignore') {
               // Special case to suggest @ts-expect-error instead of @ts-ignore
               context.report({
-                node: comment,
                 messageId: 'tsIgnoreInsteadOfExpectError',
+                node: comment,
                 suggest: [
                   {
-                    messageId: 'replaceTsIgnoreWithTsExpectError',
```

Comment:
i also think putting the messageId first and the fix afterwards is beneficial - can that be added to the sorting rule?